src/relay/op/tensor/transform.cc

```diff
@@ -315,5 +313,72 @@ Example::
 .set_support_level(3)
 .add_type_rel("Reshape", ReshapeRel);
 
+Expr MakeSqueeze(Expr data,
+                 Array<IndexExpr> axis) {
+  auto attrs = make_node<SqueezeAttrs>();
+  attrs->axis = std::move(axis);
+  static const Op& op = Op::Get("squeeze");
+  return CallNode::make(op, {data}, Attrs(attrs), {});
+}
+
+TVM_REGISTER_API("relay.op._make.squeeze")
+.set_body([](const TVMArgs& args, TVMRetValue* rv) {
+    runtime::detail::unpack_call<Expr, 2>(MakeSqueeze, args, rv);
+  });
+
+bool SqueezeRel(const Array<Type>& types,
```

Comment:
this relation could benefit from a couple high level comments about implementation